relay-server/src/actors/processor.rs

```diff
@@ -988,7 +998,10 @@ impl EnvelopeProcessorService {
             };
 
             producer.send(TrackOutcome {
-                timestamp: timestamp.as_datetime(),
+                // If we get to this point, the unwrap should not be used anymore, since we know by
+                // now that the timestamp can be parsed, but just incase we fallback to UTC current
+                // `DateTime`.
+                timestamp: timestamp.as_datetime().unwrap_or_else(Utc::now),
```

Comment:
👍 for using an unwrap with default here along with a descriptive comment. Even if the unwrap cannot fail, this line will never panic even if the rest is refactored.